framework/src/ble_manager/ble_manager_autoconnect.c

```diff
@@ -114,6 +114,10 @@ static void *_autocon_process(void *param)
 		}
 		memset(buf, 0, sizeof(buf));
 		nbytes = mq_receive(ctx->mqfd, buf, sizeof(buf), 0);
+		if (nbytes < 0) {
```

Comment:
How about to consider EAGAIN or EINTR. if those events happened then it doesn't need to finish.

Reply:
1. if returned size is smaller than size of buf, is it acceptable?
2. Instead of mq_receive, you should use mq_timedreceive..otherwise, it will wait forever when some kind of crash happened.
3. If you apply mq_timedreceive, you should handle -EPIPE case.

Reply:
@pillip8282 
Thanks. I added errno checking process.

Reply:
@Taejun-Kwon 
1. Yes. 'buf' is for getting one byte.
2. The mq will be managed by BLE_manager.
3. Yes. I will check that errno when I use mq_timedreceive.

Reply:
I think that If mq_receive return the values(eg. eintr) it should try to do mq_receive again not to go to while(). Because ble_client_start_scan is already done. Please check my comment.

Reply:
@an4967 Are there any possibilities to have nbytes = 0?

Reply:
@pillip8282 
Yes, you are right. 
I will check EINTR & EAGAIN with loop.